tests/periph_rtc/main.c

```diff
@@ -60,7 +60,7 @@ static void cb(void *arg)
 int main(void)
 {
     struct tm time = {
-        .tm_year = 2011 - TM_YEAR_OFFSET,   /* years are counted from 1900 */
+        .tm_year = 2020 - TM_YEAR_OFFSET,   /* years are counted from 1900 */
         .tm_mon  = 11,                      /* 0 = January, 11 = December */
```

Comment:
Maybe we should use a date that from today's perspective is not in the future.

Reply:
I don't see how that makes a difference, but we might as well test that nothing breaks in leap years when we change it anyway.

Reply:
I just meant that using November 13, 2020 seems a bit strange. I'm ok with it, it's just a test.